node-game2d-lib/index.js

```diff
@@ -0,0 +1,258 @@
+
+'use strict'
+
+/**
+ * Make the value -1 if less than 0 or 1 if more than 0.
+ */
+module.exports.sign = (value) => (parseFloat(value) < 0.0) ? -1.0 : 1.0;
+
+/**
+ * make a value be between the minimum and maximum
+ */
+module.exports.clamp = (value, minimum, maximum) =>
```

Comment:
My first suggestion would be to modularize the code by putting each function in its own file and then export it into the index.js file.

Reply:
You are correct. I should have modularized them better or at least some of it. I just forgot that when I was coding. I'll refactor them in the future. Thanks.